src/Extracting/Strategies/BodyParameters/GetFromFormRequest.php

```diff
@@ -66,8 +67,14 @@ public function getBodyParametersFromFormRequest(ReflectionFunctionAbstract $met
             if (
                 (class_exists(LaravelFormRequest::class) && $parameterClass->isSubclassOf(LaravelFormRequest::class))
                 || (class_exists(DingoFormRequest::class) && $parameterClass->isSubclassOf(DingoFormRequest::class))) {
-                /** @var LaravelFormRequest|DingoFormRequest $formRequest */
-                $formRequest = new $parameterClassName;
+                try {
+                    /** @var LaravelFormRequest|DingoFormRequest $formRequest */
+                    $formRequest = new $parameterClassName;
+                } catch (ArgumentCountError $e) {
+                    c::info('Skipping instanciation of ' . $parameterClassName . ' because of dependency injection. Use manual @bodyParam to describe this request.');
```

Comment:
Spelt with a _t_, not _c_.

```suggestion
                    c::info('Skipping instantiation of ' . $parameterClassName . ' because of dependency injection. Use manual @bodyParam to describe this request.');
```